pkg/platform/abstract/platform.go

```diff
@@ -221,13 +221,21 @@ func (ap *Platform) ValidateCreateFunctionOptions(createFunctionOptions *platfor
 	}
 
 	// Verify _trigger's MaxWorkers value is making sense
+	var httpTriggerExists bool
 	for triggerName, _trigger := range createFunctionOptions.FunctionConfig.Spec.Triggers {
 		if _trigger.MaxWorkers > trigger.MaxWorkersLimit {
 			return errors.Errorf("MaxWorkers value for %s trigger (%d) exceeds the limit of %d",
 				triggerName,
 				_trigger.MaxWorkers,
 				trigger.MaxWorkersLimit)
 		}
+		if _trigger.Kind == "http" {
+			if !httpTriggerExists {
+				httpTriggerExists = true
+				continue
+			}
+			return errors.New("There's more than one http trigger (unsupported)")
```

Comment:
```suggestion
			return errors.New("Only one http trigger is supported")
```

Reply:
> Only one http trigger is supported
Okay, so the one HTTP trigger _is_ supported.
But what about the reset of the HTTP triggers?
Are they _not_ supported?

😁

Reply:
lol "reset"

Reply:
What are you talking about?

Reply:
Should've print screened. You comment updater you

Reply:
![image](https://user-images.githubusercontent.com/18181802/73352659-451ac300-429a-11ea-9488-d303943f3c17.png)